extensions/taskplane/src/sign-out.tsx

```diff
@@ -0,0 +1,28 @@
+import { List, ActionPanel, Action, showToast, Toast, Icon, useNavigation } from "@raycast/api";
+import { logout } from "./lib";
+
+export default function SignOut() {
+  const { pop } = useNavigation();
+
+  return (
+    <List>
+      <List.Item
+        actions={
+          <ActionPanel>
+            <Action
+              onAction={() => {
+                logout();
+                showToast(Toast.Style.Success, "Signed out successfully");
+                pop();
+              }}
+              title="Sign out"
+            />
+          </ActionPanel>
+        }
+        title="Sign out"
+        subtitle="Sign out of your Taskplane account"
+        icon={Icon.XmarkCircle}
+      />
+    </List>
+  );
+}
```

Comment:
```suggestion
import { List, ActionPanel, Action, showToast, Toast, Icon, useNavigation } from "@raycast/api";
import { logout } from "./lib";

const SignOut = async () => {
  logout();
};
export default SignOut;
```

Reply:
It doesn't seem like `LocalStorage.removeItem` works in a `no-view` command?

Reply:
Let me check that

Reply:
Seems like there is a bug with `confirmAlert` - for now (until we fix it)

```
export async function logout() {
  await LocalStorage.removeItem("x-taskplane-token");
  showToast(Toast.Style.Success, "Logged out successfully");
}
```

Reply:
Yeah, I tried this too but it didn't work either. The toast never shows. Tried wrapping it in a try catch but nothing got logged there either.

Reply:
![CleanShot 2022-07-08 at 09 34 36@2x](https://user-images.githubusercontent.com/38033107/177941384-bf5b76fe-5d95-431f-9b22-29c49c28b87e.png)
The toast should actually work, could you give it a go more?

Reply:
Fixed